### PR TITLE
Add document.domain deprecation

### PR DESCRIPTION
**Pull request by ericlaw1979**
Preview: [https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/site-impacting-changes?branch=pr-en-us-1958](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/site-impacting-changes?branch=pr-en-us-1958)

### CODE REVIEW

**Comment by jm-trd-ms** [comment]
@ericlaw1979 
Please notify the current owners of the repo. Thank you and cheers.

**Review by vchapel — APPROVED** [review]
LGTM

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ This table lists:
...
 | Block external protocols in sandboxed frames by default | v103 | | Blocks the use of external protocols (that interact with non-browser applications) from sandboxed iframes unless permission is explicitly granted by the `sandbox` attribute on the frame. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, see the [Chrome Platform Status entry](https://chromestatus.com/feature/5680742077038592). |
+| Ignore modifications to `document.domain` by default | v106 | | The `document.domain` property historically could be set to relax Same-Origin-Policy and allow subdomains from a site to interact. This behavior will be disabled by default such that setting the `document.domain` property will have no effect. See the [Edge Reference Documentation](/deployedge/edge-learnmore-origin-keyed-agent-cluster) for more information and workarounds.
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:51]:
```suggestion
| Ignore modifications to `document.domain` by default | v106 | | The `document.domain` property historically could be set to relax Same-Origin-Policy and allow subdomains from a site to interact. This behavior will be disabled by default such that setting the `document.domain` property will have no effect.  For more information and workarounds, see [Microsoft Edge will disable modifying document.domain](/deployedge/edge-learnmore-origin-keyed-agent-cluster).
```

Comment (vchapel) [reply microsoft-edge/web-platform/site-impacting-changes.md]: fixed

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**